### ISSUE TITLE
Resolve what to print, if anything, for a copyright notice.

### ISSUE THREAD

**Issue by rkent** [ros-infrastructure/rosdoc2#77]
The copyright notice for packages is still at 2021, and flagged with a todo [here](https://github.com/ros-infrastructure/rosdoc2/blob/bccae77c43e6c6eff20c3aa1b2f77a9fd259c405/rosdoc2/verbs/build/builders/sphinx_builder.py#L229C1-L230C39).

```
# TODO(tfoote) The docs say year and author but we have this and it seems more relevant.
copyright = '2021, {package_licenses}'
```

IANAL but the subject of copyright notices has different opinions, but clearly this one (a fixed date) is not right. I would argue for:
- no date
- mention the license and authors from package.xml

Anyway this needs fixing.

**Comment by clalancette** [comment]
> Anyway this needs fixing.

We should be able to do a dynamic date by doing something similar to https://github.com/ros2/ros2_documentation/blob/84dd416a3a284758626414e183bcaada43c93c95/conf.py#L46

**Comment by rkent** [comment]
That's what I did in my demo version.

But I did some quick googling of the topic, and there seems to be an argument that 'copyright date' is supposed to represent when a documented is created (or at least released to the public). Small changes later might not be enough to update the copyright date. We also don't really know who the copyright owner is. We can't really rely on ```author``` from package.xml for that.

So I would argue that we add some statement to the effect that the document is copyrighted under such-and-such a license, without trying to give a date or author. If this is important to a package owner, they can add that themselves. Our uses are mostly going to care about which license it is under.
